### PR TITLE
add prs welcome badge

### PR DESCRIPTION
**Pull request by kentcdodds**
As referenced in #6332, this adds the `PRs | Welcome` badge to the README which links to a free beginner friendly resource teaching how to get started contributing to open source.

Here's before:

screen shot 2016-03-24 at 1 26 41 pm

Here's after:

screen shot 2016-03-24 at 1 26 33 pm

Happy to make any changes. Just want to get the conversation going.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# [React](https://facebook.github.io/react/) [![Build Status](https://img.shields.io/travis/facebook/react/master.svg?style=flat)](https://travis-ci.org/facebook/react) [![Coverage Status](https://img.shields.io/coveralls/facebook/react/master.svg?style=flat)](https://coveralls.io/github/facebook/react?branch=master) [![npm version](https://img.shields.io/npm/v/react.svg?style=flat)](https://www.npmjs.com/package/react)
+# [React](https://facebook.github.io/react/) [![Build Status](https://img.shields.io/travis/facebook/react/master.svg?style=flat)](https://travis-ci.org/facebook/react) [![Coverage Status](https://img.shields.io/coveralls/facebook/react/master.svg?style=flat)](https://coveralls.io/github/facebook/react?branch=master) [![npm version](https://img.shields.io/npm/v/react.svg?style=flat)](https://www.npmjs.com/package/react) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
```
Comment (jimfb) [inline README.md]: As mentioned in the other PR, I don't feel quite comfortable with that link / lesson plan, for reasons mentioned here: https://github.com/facebook/react/pull/6332#issuecomment-200985826

The badge its self (unlinked) could be ok.  Or we could link it to a gist that describes the React-specific notes about how we manage pull requests.

Comment (kentcdodds) [inline README.md]: I'd be cool with the badge linking to react-specific notes. I'm mostly motivated by a desire to make it easier for beginners to contribute to React.

Comment (jimfb) [inline README.md]: I'd be fine with this if we link it to https://github.com/facebook/react/blob/master/CONTRIBUTING.md for now.  We can find a different/better destination later.

**Comment by kentcdodds** [comment]
It now links to the `Pull Requests` section of the `CONTRIBUTING.md`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# [React](https://facebook.github.io/react/) [![Build Status](https://img.shields.io/travis/facebook/react/master.svg?style=flat)](https://travis-ci.org/facebook/react) [![Coverage Status](https://img.shields.io/coveralls/facebook/react/master.svg?style=flat)](https://coveralls.io/github/facebook/react?branch=master) [![npm version](https://img.shields.io/npm/v/react.svg?style=flat)](https://www.npmjs.com/package/react)
+# [React](https://facebook.github.io/react/) [![Build Status](https://img.shields.io/travis/facebook/react/master.svg?style=flat)](https://travis-ci.org/facebook/react) [![Coverage Status](https://img.shields.io/coveralls/facebook/react/master.svg?style=flat)](https://coveralls.io/github/facebook/react?branch=master) [![npm version](https://img.shields.io/npm/v/react.svg?style=flat)](https://www.npmjs.com/package/react) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](CONTRIBUTING.md#pull-requests)
```
Comment (jimfb) [inline README.md]: Should be the full path to the page: `https://github.com/facebook/react/blob/master/CONTRIBUTING.md`

Comment (kentcdodds) [inline README.md]: `¯\_(ツ)_/¯` works fine either way. Test it [here](https://github.com/kentcdodds/react/blob/pr/beginner-friendly-badge/README.md). Nice thing about this is that it'll work in forks as well. But let me know if you want me to change it.

Comment (jimfb) [inline README.md]: Yes, I think we need to change it.  I remember it being a problem and breaking links somewhere.  I think that was in the release notes which were being rendered onto a different domain.  Anyway, we should just use the absolute links everywhere, that was my takeaway from that debacle.

Comment (zpao) [inline README.md]: It's fine. The reason it broke in a changelog is because it was copied to the docs.

**Comment by zpao** [comment]
Thanks @kentcdodds!

**Comment by kentcdodds** [comment]
Fantastic!

**Pull request merged**